var/spack/repos/builtin/packages/node-js/package.py

```diff
@@ -98,12 +78,28 @@ class NodeJs(Package):
 
     # https://github.com/nodejs/node/blob/master/BUILDING.md#unix-and-macos
     depends_on("gmake@3.81:", type="build")
-    depends_on("python@3.6:3.11", when="@19.1:", type="build")
-    depends_on("python@3.6:3.10", when="@16.11:19.0", type="build")
-    depends_on("python@3.6:3.9", when="@16.0:16.10", type="build")
-    depends_on("python@2.7,3.5:3.8", when="@15", type="build")
-    depends_on("python@2.7,3.6:3.10", when="@14.18.2:14", type="build")
-    depends_on("python@2.7,3.5:3.8", when="@13.1:14.18.1", type="build")
+
+    # python requirements are based according to
+    # https://github.com/spack/spack/pull/47942#discussion_r1875624177
+    depends_on("python@:3.7", when="@v13.0.0:v13.0.1", type="build")
```

Comment:
Can you remove the "v" from these versions?

Reply:
good catch!

Reply:
see https://github.com/spack/spack/pull/47942/commits/0675f6dc4454d4d3cab4d19ac60be0c10529a875